app/src/main/java/com/github/se/orator/ui/battle/BattleRequestSentScreen.kt

```diff
@@ -85,11 +119,27 @@ fun BattleRequestSentScreen(
                       Modifier.padding(bottom = AppDimensions.paddingMedium)
                           .testTag("battleRequestSentText"))
 
+              Spacer(modifier = Modifier.height(AppDimensions.paddingLarge))
+
+              // Loading Indicator
               CircularProgressIndicator(
                   color = AppColors.loadingIndicatorColor,
                   strokeWidth = AppDimensions.strokeWidth,
                   modifier =
                       Modifier.size(AppDimensions.loadingIndicatorSize).testTag("loadingIndicator"))
+
+              Spacer(modifier = Modifier.height(AppDimensions.paddingLarge))
+
+              // Cancel Button
+              ActionButton(
+                  "Cancel Battle",
+                  onClick = {
+                    battleViewModel.cancelBattle(battleId)
+                    navigationActions.navigateTo(TopLevelDestinations.HOME)
+                    Toast.makeText(context, "Battle has been cancelled!", Toast.LENGTH_SHORT).show()
+                  },
+                  textColor = Color.White,
```

Comment:
You have hard coded colors here. Consider putting them in Color.kt.

Reply:
That is right. Good catch!